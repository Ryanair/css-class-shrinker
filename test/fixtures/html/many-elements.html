<div class="aaa-ooo"></div>
<div class="eee">
  <h1 class="iii"></h1>
  <span class="aaa-ooo"></span>
  <button class="ooo"></button>
  <div class="uuu">
    <div class="qqq"></div>
  </div>
</div>
