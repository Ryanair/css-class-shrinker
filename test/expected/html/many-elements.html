<div class="dFW"></div>
<div class="lpdD">
  <h1 class="fsG"></h1>
  <span class="dFW"></span>
  <button class="acvD"></button>
  <div class="gDs">
    <div class="Hst"></div>
  </div>
</div>
